external/artinis/ft_nirs_referencechannelsubtraction.m

```diff
@@ -0,0 +1,218 @@
+function data_rcs = ft_nirs_referencechannelsubtraction(cfg, datain)
+% FT_EXAMPLEFUNCTION demonstrates to new developers how a FieldTrip function should look like
```

Comment:
that was of course on purpose to show that I used ft_examplefunction as a template... :)
thanks for checking!